pkg/istio-agent/sds-agent.go

```diff
@@ -261,6 +261,74 @@ func gatewaySdsExists() bool {
 	return !os.IsNotExist(err)
 }
 
+// explicit code to determine the root CA to be configured in bootstrap file.
+// It may be different from the CA for the cert server - which is based on CA_ADDR
+// Replaces logic in the template:
```

Comment:
how does it replace the logic if its still there?

Reply:
Keeping the PRs small - template and bootstrap are separate.

Reply:
Added the template and related chages.